python/doc/examples/functional_modeling/vectorial_functions/plot_functions_inputDim.py

```diff
@@ -0,0 +1,136 @@
+"""
+==========================================
+Increase the input dimension of a function
+==========================================
+"""
+# %%
+# Import the required modules
+
+# %%
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a function :math:`f : \mathbb{R}^d \mapsto \mathbb{R}` from *p* functions :math:`f_i: \mathbb{R} \mapsto \mathbb{R}`.
+#
+# We can do that:
+#
+#     - Case 1: by tensorisation of the functions :math:`f_i`,
+#     - Case 2: by addition of the functions :math:`f_i`.
+#
+# We need to implement both basic steps:
+#
+#     - the projection function: :math:`\pi_i : (x_1, \dots, x_d) \mapsto x_i`,
+#     - the composed function: :math:`g_i = f_i \circ \pi_i : (x_1, \dots, x_d) \mapsto f_i(x_i)`.
+#
+#
+
+# %%
+# Basic step 1: Creation of the  projection function
+# ==================================================
+#
+# The projection function is defined by:
+#
+# .. math::
+#
+#      \pi_i : (x_1, \dots, x_d) \mapsto x_i
+#
+# We can do that using :
```

Comment:
The list is not correctly formatted:

![image](https://user-images.githubusercontent.com/31351465/230656855-cde3edd7-c03d-45d9-9384-774d22365b54.png)

Reply:
ok